api/v1_validators.inc

```diff
@@ -35,21 +39,29 @@ function validate_wordlist($wordlist, $data)
 
 function validate_page_name($pagename, $data)
 {
-    $pages = $data[":projectid"]->get_page_names_from_db();
-    if (!in_array($pagename, $pages)) {
-        throw new NotFoundError("No such page in project");
+    try {
+        $pages = $data[":projectid"]->get_page_names_from_db();
+        if (!in_array($pagename, $pages)) {
+            throw new NonexistentPageException("No such page in project");
+        }
+        return $pagename;
+    } catch (NonexistentPageException $exception) {
+        throw new NotFoundError($exception->getMessage(), $exception->getCode());
     }
-    return $pagename;
 }
 
 function validate_page_round($pageround, $data)
 {
     global $Round_for_round_id_;
 
-    $pagerounds = array_merge(["OCR"], array_keys($Round_for_round_id_));
+    try {
+        $pagerounds = array_merge(["OCR"], array_keys($Round_for_round_id_));
 
-    if (!in_array($pageround, $pagerounds)) {
-        throw new NotFoundError("Invalid page round");
+        if (!in_array($pageround, $pagerounds)) {
```

Comment:
Picky, but there's a blank line at line 59, whereas `function validate_page_name` follows the same pattern but doesn't have a blank line preceding the `if`.

Reply:
There isn't really any reason to have a blank line here.